ansible/vars/csm_packages.yml

```diff
@@ -34,7 +34,8 @@ common_csm_sles_packages:
   - csm-node-heartbeat
   - csm-node-identity
   - hpe-yq
-  - spire-agent
+  - spire-agent>=1.5.0
+  - tpm-provisioner
```

Comment:
This RPM looks like it only has an x86_84 version but no Arm version. But this package list will be applied to both x86 and Arm nodes/images. I think this is the first package being added to the list where this is the case. Because of this, this package probably needs to be handled differently and cannot just be added to this same list. Otherwise plays will fail when run on Arm nodes/images.

Reply:
This RPM is x86 only because that is what the underlying packages have. The package is only needed for TPM enablement on compute nodes. There is no failures on x86 that dont have it unless you are trying to enable spire TPM on that node.

Reply:
To clarify -- is this only needed for compute nodes? Or is TPM enablement also possible/supported for NCNs and/or application nodes? The package list it is added to here means it will go onto ALL nodes, regardless of type.

Reply:
Had a chat with Nick about this. First, he clarified that TPM enablement is supported on all node types (NCN, compute, etc) and that on a given system, it is expected to be heterogeneous where it is enabled on some nodes and not enabled on others.

Nick is going to find out whether or not Arm support is required/important for Spire TPM.

If it IS, then he will need to work to get the `tpm-provisioner` package built for Arm, and then this `csm-config` PR will work as-is.

If it is NOT, then I will help get this `csm-config` PR modified so that both the `tpm-provisioner` AND `cray-spire-dracut` RPMs (since that RPM has `tpm-provisioner` listed as a requirement) are only applied when the target architecture is x86.